srcDLL/op2ext.cpp

```diff
@@ -120,17 +120,20 @@ void GenericLog(const char* message, LogFunction logFunction)
 
 OP2EXT_API void Log(const char* message)
 {
-	GenericLog(message, LogMessage);
+	const void* moduleAddress = _ReturnAddress();
+	GenericLog(message, moduleAddress, LogMessage);
```

Comment:
Should be fine to combine these two lines and get rid of the `moduleAddress` temporary.

Actually, the name `moduleAddress` is even slightly wrong, since the address is not the base address of the module, but rather the address of the instruction after the `CALL` instruction that called into this function. That will be somewhere within the module's address space, but certainly not at the base address.

Same for the other call points.

Reply:
fixed by removing the temporary variable. I wasn't sure if mingw would be okay with calling the temporary as an argument on a function.